provision-base.sh

```diff
@@ -331,11 +366,15 @@ install_base()
 
 	if [ "$BOURNE_SHELL" = "bash" ]; then
 		install_bash
-	fi
+    elif [ "$BOURNE_SHELL" = "zsh" ]; then
+		install_zsh
+	    config_zsh_shell
+    fi
 
 	install_ssmtp
 	disable_root_password
 	install_periodic_conf
+    install_vimrc
```

Comment:
It's better to have consistent use of whitespace, (tabs-vs-spaces). Most of the MT6 scripts use tabs but my only really strong preference is consistency.